ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/CommandDeduplicationBase.scala

```diff
@@ -95,14 +95,14 @@ private[testtool] abstract class CommandDeduplicationBase(
           assertGrpcError(
             failure1,
             Status.Code.ALREADY_EXISTS,
-            "",
-            checkDefiniteAnswerMetadata = true,
+            exceptionMessageSubstring = None,
+            expectedDefiniteAnswerMetadata = Some(false),
```

Comment:
Whether a completion is a definite answer will vary from ledger to ledger. Canton, e.g., probably sets `true` here. I don't really see how we can reuse these test cases across ledgers if we test for the concrete value.

Reply:
This was my concern too. Do you want me to revert the last 2 commits?

Reply:
cc @fabiotudone-da

Reply:
Could have asked before addressing the review comments :(

Reply:
Ok, ignore, reverting the assertion changes.

Reply:
Yes, please revert the assertion change that fixes the value of `definite_answer`. Otherwise this test is unusable for Canton.

Reply:
Done, JFYI enabling the auto-merge.